website/docs/guides/airflow-quickstart.md

```diff
@@ -1,352 +1,337 @@
+2. In Marquez, navigate to the Datasets view and select your Postgres instance from the namespace dropdown menu in the top-right corner. Then, click on the `<database>.public.counts` dataset and inspect the graph. You'll find the schema on the node:
 
-![](./docs/astro-lineage-view-dataset.png)
+    ![](./docs/counts-graph-new-schema.png)
 
-Team `B` soon realizes that the schema has changed recently for the `counts` table! To fix the DAG, team `B` updates the `t2` task that calculates the count total to use the new column name:
+3. Imagine you don't recognize the column and want to know what it was originally and when it changed. Clicking on the node will open the detail drawer. There, using the version history, find the run in which the schema changed:
 
-```diff
-query2 = PostgresOperator(
-    task_id='total',
-    postgres_conn_id='example_db',
-    sql='''
-    INSERT INTO sums (value)
--       SELECT SUM(value) FROM counts;
-+       SELECT SUM(value_1_to_10) FROM counts;
-    '''
-)
-```
+    ![](./docs/counts-detail.png)
 
-With the code change, the DAG `sum` begins to run successfully:
+4. In Airflow, fix the downstream DAG that broke by updating the task that calculates the count total to use the new column name:
+
+    ```diff
+    query2 = PostgresOperator(
+        task_id='total',
+        postgres_conn_id='example_db',
+        sql='''
+    -    INSERT INTO sums (value)
+    -       SELECT SUM(value) FROM counts;
+    +       SELECT SUM(value_1_to_10) FROM counts;
+        '''
+    )
+    ```
 
-![](./docs/astro-lineage-view-job-successful.png)
+5. Rerun the DAG. In Marquez, verify the fix by looking at the recent run history in the DataOps view:
 
-_Congrats_! You successfully step through a troubleshooting scenario of a failing DAG using metadata collected with Marquez! You can now add your own DAGs to `dags/` to build more complex data lineage graphs.
+    ![](./docs/sum-history.png)
 
 ## Next Steps
 
 * Review the Marquez [HTTP API](https://marquezproject.github.io/marquez/openapi.html) used to collect Airflow DAG metadata and learn how to build your own integrations using OpenLineage.
-* Take a look at [`openlineage-spark`](https://openlineage.io/docs/integrations/spark/) integration that can be used with Airflow.
+* Take a look at the [`openlineage-spark`](https://openlineage.io/docs/integrations/spark/) integration that can be used with Airflow.
 
 ## Feedback
 
-What did you think of this example? You can reach out to us on [Slack](http://bit.ly/MqzSlack) and leave us feedback, or [open a pull request](https://github.com/MarquezProject/marquez/blob/main/CONTRIBUTING.md#submitting-a-pull-request) with your suggested changes!
\ No newline at end of file
+What did you think of this example? You can reach out to us on [Slack](http://bit.ly/MqzSlack) and leave us feedback, or [open a pull request](https://github.com/MarquezProject/marquez/blob/main/CONTRIBUTING.md#submitting-a-pull-request) with your suggested changes!
```

Comment:
should this include links to both OL and Marquez slack?

Reply:
Oops! Copy and paste error. Will fix!